plugins/services/src/js/utils/CreateServiceModalFormUtil.js

```diff
@@ -31,168 +31,6 @@ const CreateServiceModalFormUtil = {
 
       return memo;
     }, baseObject);
-  },
```

Comment:
I like we are using `jsondiffpatch` and removing the code below :) 

_(ps: have you looked this one https://github.com/kpdecker/jsdiff)_